keepassxc-browser/background/event.js

```diff
@@ -160,7 +160,7 @@ kpxcEvent.onLoginPopup = function(tab, logins) {
     browserAction.stackUnshift(stackData, tab.id);
 
     if (logins.length > 0) {
-        page.tabs[tab.id].loginList = logins[0];
+        page.tabs[tab.id].loginList = logins;
```

Comment:
Do we need to check the length? Wouldn't a return of "no logins" result in something leftover in the page's login list?

Reply:
Oh yeah, we don't. Missed this one.

Reply:
True. That function is not even called if the Array is empty.